docs/app/partials/getting-started.tmpl.html

```diff
@@ -1,38 +1,84 @@
 <div ng-controller="GuideCtrl" class="doc-content">
   <md-content>
-    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to
-      <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
-         title="Link opens in a new window">read about the framework</a>.</em></p>
+    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to:</em>
+    <ul>
+      <li>watch the videos about <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
+               title="Link opens in a new window">Angular.js framework</a></li>
```

Comment:
>
The objective of this technique is to demonstrate how to use a title attribute on an anchor element to provide additional text describing a link. The title attribute is used to provide additional information to help clarify or further describe the purpose of a link. If the supplementary information provided through the title attribute is something the user should know before following the link, such as a warning, then it should be provided in the link text rather than in the title attribute.

http://www.w3.org/TR/WCAG20-TECHS/H33.html